plugins/foreman_discovery/4.0/index.md

```diff
@@ -225,12 +225,11 @@ on Debian and Ubuntu systems.
 
 Restart of Foreman is needed when installing the package separately.
 
-## 2.2 Foreman Proxy Discovery plugin
+## 2.2 Foreman Proxy Discovery plugin (optional)
 
-Foreman Proxy Discovery plugin must be installed on all proxy hosts on the
-network where discovery feature is supposed to work. It is possible to skip
-this step if direct communication is planned, but configuration must be
-changed in that case.
+Foreman Proxy plugin must be only installed when Foreman server does not have
+direct connection to. Foreman Proxy Discovery plugin must be installed on all
```

Comment:
> Foreman Proxy plugin must be only installed when Foreman server does not have direct connection to. 

Maybe add "to the discovered host."  

Or rephrase, "Smart Proxy plugin can be used to discover hosts in networks Foreman can't directly reach itself."